tests/merchOrder.test.ts

```diff
@@ -590,6 +590,44 @@ describe('merch orders', () => {
       .rejects.toThrow(`Not allowed to order: ${[option.uuid]}`);
   });
 
+  test('order route accurately verifies monthly and lifetime limits', async () => {
+    const conn = await DatabaseConnection.get();
+    const optionMetadataType = faker.datatype.hexaDecimal(10);
+    const option1 = MerchFactory.fakeOptionWithType(optionMetadataType);
+    const option2 = MerchFactory.fakeOptionWithType(optionMetadataType);
+    const item = MerchFactory.fakeItem({
+      options: [option1, option2],
+      monthlyLimit: 1,
+      lifetimeLimit: 1,
+    });
+    const orderPickupEvent = MerchFactory.fakeFutureOrderPickupEvent();
+    const member = UserFactory.fake({
+      accessType: UserAccessType.STANDARD,
+      credits: 1 * option1.price + 1 * option2.price,
+    });
```

Comment:
Why do we have `(1 * option1.price)` and `(1 * option2.price)` here? I think credits should be `(100 * option.price)`.

Reply:
Are the option prices and user credits stored as different scales? The "1 *" was leftover from some experimentation and can be removed

Reply:
Also, just checked, in either case, the limit is checked before the credits so this shouldn't be an issue unless the limit fires first

Reply:
Oh my bad, yeah credits are the same as price so we can omit the 1.